python/src/LinearTaylor_doc.i.in

```diff
@@ -7,41 +7,27 @@ Available constructors:
 Parameters
 ----------
 center : sequence of float
-    Point :math:`\vect{x}_0` where the Taylor expansion of the function
-    :math:`h` is performed.
+    Point :math:`\ux_0`.
 function : :class:`~openturns.Function`
-    Function :math:`h` to be approximated.
+    Function :math:`h` to be approximated around :math:`\ux_0`.
```

Comment:
Not "_around_". The Taylor expansion is created _"at"_ a single point.
```suggestion
    Function :math:`h` to be approximated at the point :math:`\ux_0`.
```

Reply:
yes...